scripts/apps/authoring/views/article-edit.html

```diff
@@ -501,7 +512,7 @@
     sd-width="{{ editor[field._id].sdWidth || 'full' }}"
     data-required="schema[field._id].required"
     order="{{ editor[field._id].order }}"
-    sd-validation-error="error.extra[field._id]">
+    sd-validation-error="error[field._id.toLowerCase()]">
```

Comment:
If field._id has the same value as the name of a field from the item schema (e.g. headline) this will create confusion.

Reply:
The errors are added in a dictionary by id lowercase. Why is this a problem?

Reply:
Say you have a vocabulary with id 'headline'. If you got an error on the vocabulary field it will look as an error on the 'headline' field from the item. I know it's an edge case, but why not keep custom vocabulary errors in extra?

Reply:
When it comes from backend I don't know if it is an custom extra (custom vocabulary or field) error or not. This is more a vocabulary and custom field validation.